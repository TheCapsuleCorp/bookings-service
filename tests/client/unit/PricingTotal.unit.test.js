import React from 'react';
import { configure } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import { shallow } from 'enzyme';
import { shallowToJson } from 'enzyme-to-json';
import PricingTotal from '../../../client/src/components/PricingTotal.jsx';
import bookingsSample from '../../../database/dummy_data.js';

configure({ adapter: new Adapter() });

describe('PricingTotal', () => {
  it('should render correctly', () => {
    const output = shallow(
      <PricingTotal listing={bookingsSample}></PricingTotal>
    );
    expect(shallowToJson(output)).toMatchSnapshot();
  });
});
