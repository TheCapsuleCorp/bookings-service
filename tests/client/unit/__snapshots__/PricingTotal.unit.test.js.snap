// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PricingTotal should render correctly 1`] = `""`;
